src/components/shared/Pagination/Pagination.js

```diff
@@ -9,7 +9,12 @@ export const Pagination = ({ currentPage, pagesCount, urlToGo }) => {
       <ul>
         {pageNumbers.map((number) => (
           <li key={number}>
-            <Link to={`${urlToGo}page=${number}`}>{number}</Link>
+            <Link
+              className={number === Number(currentPage) ? 'dp-active-page' : ''}
```

Comment:
Why `currentPage` is not a number at this point?

Reply:
because when pass from component is a string